.gitignore

```diff
@@ -0,0 +1,2 @@
+*.o
```

Comment:
What about also ignoring the generated binaries (sgp30/sgp30_example_usage_hw_i2c, etc.)?

Reply:
Also fixed